### DISCUSSION TITLE
copy annotation link to url book section

### DISCUSSION THREAD

**Discussion by neybaf** [windingwind/zotero-actions-tags#181] [Ideas]
Hi, 
I'm trying to figure out if it would be possible to automatically copy the link from a highlight to a book section url

My workflow right now is the following:
1. Create a book section from a book
2. Open the book 
3. Copy/paste chapter's title to book section title
4. Highlight Chapter title in purple
5. Copy/paste annotation's link to book section url

Not sure if there is a quick/easy way to do this

**Comment by windingwind** [comment]
Sorry, I don't quite understand you. What is a `book section` (annotation/note?)

**Reply by neybaf** [reply]
Book section is a kind of item in Zotero that you can create from a book, a book chapter (I think it's 书章)

**Comment by mjthoraval** [comment]
> 5. Copy/paste annotation's link to book section url

@neybaf: How do you do that step at the moment?
Is that the step where you would like to create a shortcut from this plugin?

I understand that you have a `Book` item, with a PDF attachment of the full book.
Then you create a `Book Section` item from that book through: right-click on the `Book` item -> `Create Book Section`.
You then want to add a link from the `Book Section` item to the corresponding location in the PDF attachment of the `Book` item, through a link to a `Highlight` on the title of the chapter in the PDF file.

So I understand that your request would be to have a shortcut to copy an annotation link inside the PDF reader. Is it correct?

**Reply by neybaf** [reply]
Yes, I would like to have this one as shortcut. The best would be a shortcut that do the whole job for me like but I think it goes further than what is possible.

At the moment, I Highlight each title, export them to a note and then copy/paste in obsidian, clean the markdown and copy/paste the zotero link in the url section. 

The result is something like that: 

url : zotero://open-pdf/library/items/8WTA2JYC?page=599&annotation=CWDQY2G3
![Capture d’écran 2023-11-07 à 00 45 26](https://github.com/windingwind/zotero-actions-tags/assets/103755582/e04700b8-5abc-4f8e-a3be-70f328dccc27)

**Comment by windingwind** [comment]
#204
